Building docs fails with Handler <function _builder_inited at 0x7f81531a88b0> for event 'builder-inited' threw an exception

```
python -m nox -e docs && (cd docs/_build/html; python -m webbrowser localhost:8000; python -m http.server; cd -)
nox > Running session docs
nox > Re-using existing virtual environment at .nox/docs.
nox > python -m pip install --progress-bar=off wheel
nox > python -m pip install --progress-bar=off 'salt>=3003'
nox > python -m pip install -e '.[docs]'
nox > make clean
nox > make linkcheck SPHINXOPTS=-W
Running Sphinx v5.3.0
Initializing Spelling Checker 7.7.0
making output directory... done

Extension error (furo):
Handler <function _builder_inited at 0x7f81531a88b0> for event 'builder-inited' threw an exception (exception: Did you list 'furo' in the `extensions` in conf.py? If so, please remove it. Furo does not work with non-HTML builders and specifying it as an `html_theme` is sufficient.)
make: *** [Makefile:20: linkcheck] Error 2
nox > Command make linkcheck SPHINXOPTS=-W failed with exit code 2
nox > Session docs failed.
```

--- a/src/saltext/cli/project/docs/conf.py.j2
+++ b/src/saltext/cli/project/docs/conf.py.j2
@@ -66,7 +66,6 @@ rst_prolog = """
 # extensions coming with Sphinx (named 'sphinx.ext.*') or your custom
 # ones.
 extensions = [
-    "furo",
     "sphinx.ext.autodoc",
     "sphinx.ext.autosummary",
     "sphinx.ext.napoleon",

